.github/workflows/package.yaml

```diff
@@ -0,0 +1,36 @@
+name: packaging
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  macos:
+
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    env:
+      PYTHON_VERSION: ${{ matrix.python-version }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: conda-incubator/setup-miniconda@v1
+      with:
+        auto-update-conda: true
+        python-version: ${{ matrix.python-version }}
+        activate-environment: omnisci-ds-installer
+        channels: conda-forge
+
+    - name: install dependencies
+      shell: bash -l {0}
+      run: conda install constructor
```

Comment:
I think you might need to grab the conda-standalone with wget, as stated in the readme

Reply:
oh I see .. sorry .. I miss understood this part, I thought it was required just for windows. I am fixing that now. thanks @scopatz !